hw/xbox/nv2a.c

```diff
@@ -1309,11 +1311,14 @@ typedef struct VertexAttribute {
     unsigned int converted_size;
     unsigned int converted_count;
 
+    float *inline_buffer_buffer;
```

Comment:
Okay, I thought inline_buffer_buffer to make sure it's not confused with inline_array. I'll also rename gl_inline_buffer_buffer then.